recipes/mlsauce/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "mlsauce" %}
+{% set version = "0.8.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Techtonique/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: f6c3adbc6215571c4b893b9011211fcbc5d55b3739da85a03e91669735f63009
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py < 35]
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - Cython >=0.29.21
+    - matplotlib-base
+    - numpy >=1.14.0
```

Always list numpy without a constraint in host if you compile something
```suggestion
    - numpy
```